integration-test/jms-queue-service/pom.xml

```diff
@@ -62,6 +62,11 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>dk.dbc</groupId>
+      <artifactId>artemis-jms-base-image</artifactId>
+      <version>java11-SNAPSHOT</version>
+    </dependency>
```

Comment:
Vi laver den om til en pom (istedet for jar) dependency.

Reply:
Det kunne maven ikke lide, så det gør vi ikke.